### PR TITLE
Allow disabling rules

### PR DESCRIPTION
**Pull request by Hwesta**
Add 'disable' link for the models that support it that didn't have it already.

Misty had prototyped something generic to swap enabled-ness, but it didn't quite work so I copied the existing disable pattern for FPRules.

This branch currently doesn't let you re-enable something that's been disabled.

The FPR GUI code would benefit greatly from [class-based views](https://docs.djangoproject.com/en/dev/topics/class-based-views/) and templates that could accept generic classes+some class info.

Edit: Added enabling disabled rules, and fixed enable/disabled text.

### CODE REVIEW

**Comment by jhsimpson** [comment]
:+1:  

Haven't tested it yet, though.  I will set this up to test over the next few days.

**Comment by mistydemeo** [comment]
After disabling a rule, the page correctly shows the banner reading "You are viewing a disabled revision.", but the "actions" page still shows a "disable" button instead of an "enable" button. This still maps to `/fprule/:uuid/delete/`, as well.

screen shot 2016-01-28 at 11 55 43 am

**Comment by mistydemeo** [comment]
And, testing, the "disable" button doesn't toggle the rule; it remains disabled. The rule is also shown in the rules table with no indication it's disabled, and has a "disable" button instead of an "enable" button:

screen shot 2016-01-28 at 11 57 26 am

File: fpr/models.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ class Enabled(models.Manager):
     def get_queryset(self):
         return super(Enabled, self).get_queryset().filter(enabled=True)
 
+    def get_query_set(self):
+        return super(Enabled, self).get_query_set().filter(enabled=True)
```
Comment (Hwesta) [inline fpr/models.py]: I'll remove this when it gets merged for Archivematica 1.5 - it was added to support Django 1.5/Archivematica 1.4.

Comment (sevein) [inline fpr/models.py]: Makes sense!

**Comment by sevein** [comment]
I've found a bit confusing that the word `delete` is still part of the URL but other than that it seems to work great. I've done a couple of tests, made changes in the preservation planning tab and verified the changes in Transfer. 👍

**Pull request merged**